flake.nix

```diff
@@ -925,6 +926,16 @@
                     >&2 echo " ️  Please add 'set -g default-command \"\''${SHELL}\"' to your '$HOME/.tmux.conf' for tmuxinator test setup to work correctly"
                   fi
                 fi
+
+                # if runing in direnv
+                if [ -n "''${DIRENV_IN_ENVRC:-}" ]; then
+                  # and not set DIRENV_LOG_FORMAT
+                  if [ -n "''${DIRENV_LOG_FORMAT:-}" ]; then
+                    >&2 echo "💡 Set 'DIRENV_LOG_FORMAT=\"\"' in your shell scripts for cleaner output"
```

Comment:
just a clarification for myself: where is this used? It just controls what `direnv` logs when starting, right? 

```sh
$ set -x DIRENV_LOG_FORMAT 'THINGY: %s'
$ direnv allow
THINGY: loading ~/dev/mxz/fm/.envrc
THINGY: using flake
warning: ignoring untrusted substituter 'https://fedimint.cachix.org'
💡 Set 'DIRENV_LOG_FORMAT=""' in your shell scripts for cleaner output
💡 Run 'just' for a list of available 'just ...' helper recipies
THINGY: export +AR +AR_FOR_TAR[...]
```

vs an empty `set -x DIRENV_LOG_FORMAT ''` 
```sh
warning: ignoring untrusted substituter 'https://fedimint.cachix.org'
💡 Run 'just' for a list of available 'just ...' helper recipies
```

Reply:
Example. It cleans up 4-6 lines of output each time `direnv` is re-evaluating environment to use after `flake.nix` changed / you enter directory.

Reply:
By getting rid of it, I'm hoping that the other notice messages will be more visible.

Reply:
I like it. Done.